sdk/Cargo.toml

```diff
@@ -0,0 +1,332 @@
+[profile.release]
+split-debuginfo = "unpacked"
+lto = "thin"
+
+[workspace]
+members = [
```

Comment:
thoughts on making an `sdk/crates` dir so everything can just be moved in there and this field can be `members = ["crates/*"]`?

Reply:
Personally, I'm going to get way more annoyed about having to type `crates` (or `c<TAB>`) every single time I want to open a file in the repo over having to update this field whenever a new crate is added.

But I might be in the minority.